src/messages/MessageBuilder.cpp

```diff
@@ -193,6 +193,12 @@ std::pair<MessagePtr, MessagePtr> makeAutomodMessage(
 
     //
     // Builder for offender's message
+    builder2.message().channelName = channelName;
+    builder2
+        .emplace<TextElement>("#" + channelName,
+                              MessageElementFlag::ChannelName,
+                              MessageColor::System)
+        ->setLink({Link::JumpToChannel, channelName});
```

Comment:
Wouldn't this cause the channel name to be shown in the message added to the regular channel?

Reply:
no

Reply:
yea nevermind